flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -276,7 +260,7 @@ protected <K> CheckpointableKeyedStateBackend<K> restoreKeyedBackend(
                         new CloseableRegistry());
     }
 
-    @Test
+    @TestTemplate
     public void testEnableStateLatencyTracking() throws Exception {
```

Comment:
```suggestion
    void testEnableStateLatencyTracking() throws Exception {
```
could be package private